VAMobile/src/components/Nametag/Nametag.tsx

```diff
@@ -64,54 +74,144 @@ export const Nametag: FC = () => {
-                />
-              </Box>
-            )}
+              )}
+              {accessToMilitaryInfo && showVeteranStatus && (
+                <Box flexDirection={'row'} alignItems={'center'} mt={theme.dimensions.standardMarginBetween}>
+                  <TextView variant="MobileBody" color="primaryContrast" mr={theme.dimensions.textIconMargin}>
+                    {t('veteranStatus.proofOf')}
+                  </TextView>
+                  <VAIcon
+                    name={'ChevronRight'}
+                    fill={theme.colors.icon.contrast}
+                    width={theme.dimensions.chevronListItemWidth}
+                    height={theme.dimensions.chevronListItemHeight}
+                    mr={theme.dimensions.listItemDecoratorMarginLeft}
+                  />
+                </Box>
+              )}
+            </Box>
           </Box>
         </Box>
+      </Pressable>
+    )
+  } else if (homeScreen) {
+    return (
+      <Box>
+        <Box>
+          {accessToMilitaryInfo && (
+            <Pressable {...pressableProps}>
+              <Box
+                pl={theme.dimensions.buttonPadding}
+                backgroundColor={theme.colors.background.veteranStatusHome as BackgroundVariant}
+                minHeight={82}
+                display="flex"
+                justifyContent="center"
+                mb={theme.dimensions.standardMarginBetween}
+                pr={theme.dimensions.buttonPadding}
+                mx={theme.dimensions.condensedMarginBetween}
+                borderRadius={8}>
+                <Box py={theme.dimensions.buttonPadding} display="flex" flexDirection="row" alignItems="center">
+                  {getBranchSeal()}
+                  <Box ml={theme.dimensions.buttonPadding} flex={1}>
+                    <TextView variant={'VeteranStatusBranch'} pb={4}>
+                      {branch}
+                    </TextView>
+                    {showVeteranStatus && (
+                      <Box flexDirection={'row'} alignItems={'center'}>
+                        <TextView variant={'VeteranStatusProof'} mr={theme.dimensions.textIconMargin}>
+                          {t('veteranStatus.proofOf')}
+                        </TextView>
+                      </Box>
+                    )}
+                  </Box>
+                  {showVeteranStatus && (
+                    <VAIcon
+                      name={'ChevronRight'}
+                      fill={theme.colors.icon.veteranStatusHomeArrow}
+                      width={theme.dimensions.chevronListItemWidth}
+                      height={theme.dimensions.chevronListItemHeight}
+                      ml={theme.dimensions.listItemDecoratorMarginLeft}
+                    />
+                  )}
+                </Box>
+              </Box>
+            </Pressable>
+          )}
+        </Box>
+        <Box backgroundColor={theme.colors.background.veteranStatusHome as BackgroundVariant}>
+          {disRating && (
```

Comment:
After giving this some thought, I think it'd be cleaner to have the disability rating and monthly compensation directly on the home screen instead of this component. It feels like this component is handling more than it should. We could also end up adding more info to the "About you" section in the future, and it would get crammed in here